federatedgroups/lib/FederatedFileSharing/FederatedShareProvider.php

```diff
@@ -164,7 +164,7 @@ public function create(IShare $share) {
 		/*
 		 * Check if file is not already shared with the remote user
 		 */
-		$alreadyShared = $this->getSharedWith($shareWith, self::SHARE_TYPE_REMOTE, $share->getNode(), 1, 0);
```

Comment:
it seems that we should check both types `share_type_remote` and `share_type_remote_group`.
I have no idea but what if we share a file with somebody twice? once share with his user directly and again for the group that he has its membership?

Reply:
I think we still want to create the share to the group in that case.